deployment/tasks/deploy_to_server.yml

```diff
@@ -29,6 +29,9 @@
 - name: Create var/cache dir
   file: path={{ app_dir }}/{{ release_prefix }}{{ release_name }}/var/cache owner={{app_owner}} group={{app_group}} state=directory mode=775
 
+- name: Create cache-busting prefix
+  copy: dest={{ app_dir }}/{{ release_prefix }}{{ release_name }}/var/file_prefix.txt content="{{ release_name|hash('md5') }}" owner={{app_owner}} group={{app_group}}
```

Comment:
`release_name` is a variable passed in from the "parent" playbook `deployment.yml`. It contains a timestamp in format YYYYMMDDHHMMSS. 

There is not much manual to read, I've tried to improve that situation with #684